casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -72,8 +76,9 @@ object BlockAPI {
         tupleSpaceHash = PrettyPrinter.buildStringNoLimit(tsHash),
         tupleSpaceDump = tsDesc,
         timestamp = timestamp,
+        faultTolerance = normalizedFaultTolerance + initialFault,
```

Comment:
I don't understand why we are adding these two numbers.

Reply:
Oh good catch. It needs to subtract.

Reply:
Interesting. The CasperTFG paper has it flipped too.